android/app/build.gradle

```diff
@@ -148,8 +148,8 @@ android {
         applicationId "me.rainbow"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 216
-        versionName "1.9.27"
+        versionCode 218
```

Comment:
I had to bump version to 218 as we submitted multiple times to playstore leaving off at 217!